lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -77,8 +77,8 @@ def test_global_contourf_wrap_no_transform():
 
 
 global_pcolor_wrap = ('global_pcolor_wrap'
-                      if mpl.__version__ >= '1.5' else
```

Comment:
String compare is not version compare : see also [above](https://github.com/SciTools/cartopy/pull/960/files#r153180788)

Reply:
So I think you can just import `cartopy.tests.mpl.MPL_VERSION` to do this right..

Reply:
Done

Reply:
... Oh, you already did !